src/cairo_runner.rs

```diff
@@ -167,6 +171,29 @@ impl PyCairoRunner {
         write_output(&mut self.inner, &mut self.pyvm.vm.borrow_mut()).map_err(to_py_error)
     }
 
+    pub fn add(&self) -> PyRelocatable {
```

Comment:
A better name for this method would be `add_segment(),` it is a bit confusing otherwise